packages/storybook/config/storybook/jetPieThemeDark.js

```diff
@@ -1,39 +1,39 @@
-import { create } from '@storybook/theming/create';
+import { create, color, typography } from '@storybook/theming';
 import logo from '../../public/logo--pie.svg';
 
 export default create({
     base: 'dark',
 
-    colorPrimary: '#fff',
-    colorSecondary: '#2a3846',
+    // Storybook-specific color palette
+    colorPrimary: '#f36d00',
+    colorSecondary: '#4996FD',
 
     // UI
-    appBg: '#0c151d',
-    appContentBg: '#fff',
-    appBorderColor: '#c5ccd3',
-    // appBorderRadius: 0
+    appBg: '#162028',
+    appContentBg: '#1b252e',
+    appBorderColor: 'rgba(255,255,255,.1)',
+    appBorderRadius: 4,
 
-    // Typography
+    // Fonts
     fontBase: '"JustEatBasis", Arial, sans-serif',
```

Comment:
Should we change this one to `JET Sans` at some point?

Reply:
This'll switch over when we move the website over to Icing with the font changes – but worth remembering for that 😄